### PR TITLE
Add --to-saved-search-index to change index for saved search

### PR DESCRIPTION
**Pull request by wader**

### CODE REVIEW

**Comment by wader** [comment]
Not sure if a bit too specialized or could be useful to others

File: copy-kibana-dashboard.rb [hunk 1]
```diff
@@ -72,6 +76,7 @@ def parse_args
     ['--to-host', GetoptLong::OPTIONAL_ARGUMENT],
     ['--to-port', GetoptLong::OPTIONAL_ARGUMENT],
     ['--to-index', GetoptLong::OPTIONAL_ARGUMENT],
+    ['--to-saved-search-index', GetoptLong::REQUIRED_ARGUMENT],
```
Comment (jim-davis) [inline copy-kibana-dashboard.rb:79]: Why is this a required argument, not optional?

Comment (wader) [inline copy-kibana-dashboard.rb:79]: Wops good catch, will fix

Comment (wader) [inline copy-kibana-dashboard.rb:79]: Or what does optional mean here? the whole option is optional or the argument to the option is optional? i can test tomorrow

Comment (wader) [inline copy-kibana-dashboard.rb:79]: REQUIRED_ARGUMENT seems to mean that the option requires and argument. So i guess some of the other options should be REQUIRED_ARGUMENT also, or?

**Comment by wader** [comment]
Switched back to required

**Pull request merged**